dotCMS/src/main/java/com/dotcms/job/system/event/AbstractJobDelegate.java

```diff
@@ -40,8 +42,6 @@ public void execute(JobDelegateDataBean data) {
 				HibernateUtil.closeSession();
```

Comment:
shouldn't be removed this one HibernateUtil.closeSession(); too?

Reply:
yeah this one needs to go also

Reply:
the entire `finally` block can me removed